oci8_sql_test.go

```diff
@@ -1534,3 +1566,42 @@ func BenchmarkPrefetchR1000M0(b *testing.B) {
 		benchmarkPrefetchSelect(b, 1000, 0, &n)
 	}
 }
+
+type fataler interface {
```

Comment:
I don't think this is needed. Did you see testing type TB? https://golang.org/pkg/testing/#TB

Reply:
Ah, I wasn't aware of that. Thanks for the pointer